lib/c-api/tests/.gitignore

```diff
@@ -1,6 +1,11 @@
 # ignore wasm-c-api binaries
 wasm-c-api-*
 test-*
+wasm-c-api/example/*
```

Comment:
Is this change related to musl?

Reply:
nope.

It can be extracted to a dedicated PR if needed.